tests/integration/frameworks/test_tensorflow_impl.py

```diff
@@ -53,7 +54,6 @@ def cnn_model_fn():
 
         return {"p": p, "loss": loss, "train_op": train_op, "X": X, "y": y}
 
-    tf.compat.v1.reset_default_graph()
```

Comment:
why we originally need this line? @aarnphm

Reply:
I think it is due to having default graph initialized elsewhere.